torch/csrc/jit/fuser/compiler.cpp

```diff
@@ -332,10 +332,6 @@ std::shared_ptr<FusedKernel> compileKernel(
     }
   }
 
-  // Have checked the limit at graph_fuser. Assert nothing else changing that.
-  AT_ASSERT((flat_inputs.size() + flat_outputs.size()) <=
```

Comment:
Can you elaborate more about the motivation? with this change fusion compiler could not catch the case where it reach over the arguments limit (although it's rare).

Reply:
we would like to extend the arguments limit here. Not sure if this assertion should be removed.

Reply:
Then I think we can still leave the `fusion_kernel_args_limit = 128` in the header with no const expr, and set the limit in graph fuser? that way the fusion compiler can still catch the overflow on argument size.

Reply:
the limit is somewhat of a misnomer -- it's actually an NVRTC specific limitation.  Ideally it wouldn't exist at all.

I'm not sure the best approach to ripping it out, but I don't see why the compiler.cpp file would need to have this logic if it's already checked in the fusion pass